Support/Gemfile

```diff
@@ -1,3 +1,3 @@
 source "http://rubygems.org"
 
-gem "rspec", '~> 3.3'
+gem "rspec", ['~> 3.3', '< 3.5']
```

Comment:
I got an error related to the absence of the `textmate` formatter, and jumped immediately to the gemfile with that change. But I can't reproduce it so maybe it was something else.

(reverted)